dashboard/app/static/coverage.js

```diff
@@ -39,9 +39,9 @@ function initUpdateForm(){
 // "#file-details-prev" and "#file-details-curr" are the corresponding <div>s used to show per-file details.
 function onShowFileContent(url) {
   var curUrlParams = new URLSearchParams(window.location.search);
-  url += '&subsystem=' + curUrlParams.get('subsystem')
-  url += '&manager=' + curUrlParams.get('manager')
-  url += '&unique-only=' + curUrlParams.get('unique-only')
+  url += '&subsystem=' + (curUrlParams.get('subsystem') || "")
```

Comment:
Don't you also want to call `encodeURIComponent` here?

Reply:
Checked the [documentation](https://www.w3schools.com/jsref/jsref_encodeuricomponent.asp).
`, / ? : @ & = + $ #` are not expected in manager or subsystem name.

Reply:
We can use `-` there but it should't be a problem.